docs/addons/calico.md

```diff
@@ -0,0 +1,190 @@
+# Calico Add-on
+
+The `Calico` addon adds support for [Calico CNI](https://docs.projectcalico.org/about/about-calico) to an EKS cluster.
```

Comment:
We should state that it adds support for Kubernetes Network Policies as opposed to CNI. Calico is never installed as full CNI (vpc-cni is used) and it is only leveraged as a network policy engine.

Reply:
Revised.